### ISSUE TITLE
debug panel throw ErrorException

### ISSUE THREAD

**Issue by pigochu** [yiisoft/yii2-httpclient#229]
### What steps will reproduce the problem?
My Config
```php
if (!YII_ENV_TEST) {
    // configuration adjustments for 'dev' environment
    $config['bootstrap'][] = 'debug';
    $config['modules']['debug'] = [
        'class' => \yii\debug\Module::class,
        'panels' => [
            'httpclient' => [
                'class' => 'yii\\httpclient\\debug\\HttpClientPanel',
            ],
        ],
        'allowedIPs' => ['*'],
    ];
}
```

### What do you get instead?
PHP Deprecated Warning – [yii\base\ErrorException](https://www.yiiframework.com/doc-2.0/yii-base-errorexception.html)

![cut2](https://user-images.githubusercontent.com/1013744/227084583-9721140b-80cf-49bc-b951-991d5c0c3396.PNG)

![cut](https://user-images.githubusercontent.com/1013744/227084109-95d7397b-84e8-4aaa-84c8-ec0d59eabd57.PNG)

### Additional info

| Q                       | A
| ----------------------- | ---
| Yii version             |  2.0.47
| Yii HTTP Client version | 2.0.14
| PHP version             | 8.1 (docker yii2)
| Operating system        |  Linux (docker yii2)

**Comment by samdark** [comment]
Do you have time for a fix / pull request?

**Comment by samdark** [comment]
Oh, wait. Is should be already fixed in #226.
